CONTRIBUTING.md

```diff
@@ -0,0 +1,194 @@
+# Contributing
+
+Thanks for taking the time to join our community and start contributing.
+These guidelines will help you get started with the Contour project.
+Please note that we require [DCO sign off](#dco-sign-off).  
+
+## Building from source
+
+This section describes how to build Contour from source.
```

Comment:
s/Contour/ir2proxy/

Reply:
Thanks, there were a couple of other ones too, fixed now.